src/jit/emitarm64.cpp

```diff
@@ -2864,6 +2864,8 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
         case EA_8BYTE:
             result = INS_OPTS_1D;
             break;
+        default:
```

Comment:
Should check which of the default cases should be reachable and mark those that shouldn't with `unreached()` as per the JIT coding guidelines https://github.com/dotnet/coreclr/blob/master/Documentation/clr-jit-coding-conventions.md#12.4